x-pack/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card.tsx

```diff
@@ -63,11 +65,15 @@ export const ArtifactEntryCard = memo<ArtifactEntryCardProps>(
 
           <EuiSpacer size="m" />
 
-          <EuiText>
-            <p data-test-subj={getTestId('description')}>
-              {artifact.description || getEmptyValue()}
-            </p>
-          </EuiText>
+          {isTrustedApp(item as AnyArtifact) ? (
+            <EuiText>
+              <p data-test-subj={getTestId('description')}>
+                {artifact.description || getEmptyValue()}
+              </p>
+            </EuiText>
+          ) : (
+            <CardComments comments={artifact.comments} />
```

Comment:
As this component just displays the show/hide comments button if there is a comment, I'm thinking if we can just render it always and just let the condition for the description (if trustedApp then display description else null). Toughs?